### DISCUSSION TITLE
Will an update to the stable version be implemented?

### DISCUSSION THREAD

**Discussion by evilaliv3** [nextcloud/all-in-one#502] [Questions] [answered]
Hello @szaimen! Thank you for your work.

I'm curious to know if there is any plan to implement an update to the stable version guaranteeing at least full migration of the data that is stored inside next cloud or if you consider that due to the implementation you would like to complete it is likely that migration could not be possible.

**Comment by szaimen** [comment, marked as answer]
Hi, if you used the `latest` or `latest-arm64` tag when creating the mastercontainer you will automatically offered the update to the stable version 1.0.1 as soon as it gets released. This should happen in the next 7 days.

**Reply by szaimen** [reply]
done with https://github.com/nextcloud/all-in-one/releases/tag/v1.0.1
